docs/layout/partials/header.html

```diff
@@ -316,71 +327,78 @@ <h3 id="request-response">Request/Response stubbing and mocking</h3>
       <code>
         <pre>
           <span class="white">
-            > intercept("https://www.google-analytics.com/analytics.js");
+          intercept("https://www.google-analytics.com/analytics.js");
           </span>
         </pre>
       </code>
     </div>
-    <p>
-      Or redirecting an XHR request on the page to a test instance
+    <p>or redirecting an XHR request on the page to a test instance
     </p>
     <div class="code-section">
       <code>
         <pre>
           <span class="white">
-            > intercept(“https://fetchdata.com”, “http://fetchtestdata.com”)
+          intercept(“https://fetchdata.com”, “http://fetchtestdata.com”)
           </span>
         </pre>
       </code>
     </div>
     <p>
-      Or stubbing an XHR request to return custom data
+    stubbing an XHR request to return custom data
     </p>
     <div class="code-section">
       <code>
         <pre>
           <span class="white">
-            > intercept(“https://fetchdata.com”, {“test”: data})
+          intercept(“https://fetchdata.com”, {“test”: data})
           </span>
         </pre>
       </code>
     </div>
     <p>
-        Or modify data sent by the XHR request
+     modify data sent by the XHR request
     </p>
     <div class="code-section">
       <code>
         <pre> 
           <span class="white">
-            > intercept(“https://fetchdata.com”, intercept(url,(request) => 
+          intercept(“https://fetchdata.com”, intercept(url,(request) => 
```

Comment:
`intercept(“https://fetchdata.com”, intercept(url,(request) => ` should be `intercept(“https://fetchdata.com”, (request) =>`

Reply:
Thanks @NivedhaSenthil . Made this change.